resources/metadata/kubernetesdeploy.yaml

```diff
@@ -70,7 +70,7 @@ spec:
         aliases:
           - name: k8sAppTemplate
         type: string
-        description: Defines the filename for the kubernetes app template (e.g. k8s_apptemplate.yaml)
+        description: 'Defines the filename for the kubernetes app template (e.g. k8s_apptemplate.yaml). Within this file `image` needs to be set as `image: <image-name>` for the image to be overwritten with other parameters.'
```

Comment:
I'm not sure if the ` characters will work when within single quotes and I'm not sure how to check the documentation formatting changes since [the file just shows ${docGenParameters}](https://github.com/SAP/jenkins-library/blob/master/documentation/docs/steps/kubernetesDeploy.md#docgenparameters). @o-liver do you know how I can view it?

Reply:
I've never managed to review the documentation before it was in master. You can build it. I'm very sure that this will not be an issue here. The single quotes are not added in the generated go code. I'm not sure, however, if the yaml or go file is used for documentation generation. Probably the yaml. 
I would suggest merging it and then fixing it with another PR if it is a problem, instead of trying to build the documentation before hand.